src/sentry/api/exceptions.py

```diff
@@ -56,6 +58,9 @@ class SsoRequired(SentryAPIException):
 
     def __init__(self, organization, after_login_redirect=None):
         login_url = reverse("sentry-auth-organization", args=[organization.slug])
+        request = env.request
+        if request and is_using_customer_domain(request):
+            login_url = organization.absolute_url(login_url)
```

Comment:
Yea we need to do this. 

For example if the superuser is attempting to access `markstory.sentry.io`, and needs to re-authenticate via SSO, then they would be redirected to `loginUrl=sentry.sentry.io/auth/login/sentry/` to re-authenticate via `sentry`'s SSO.

Otherwise, `loginUrl=markstory.sentry.io/auth/login/sentry/` would have been used; which is incorrect.